app/models/arm.rb

```diff
@@ -27,6 +27,7 @@ class Arm < ApplicationRecord
 
   has_many :line_items_visits, :dependent => :destroy
   has_many :line_items, :through => :line_items_visits
```

Comment:
Added this to work with eager loading calls elsewhere. The definition of the eager load has to match the chains of associations being called.